### ISSUE TITLE
Contribute fix for #11943 to 0.26

### ISSUE THREAD

**Issue by 0xdaryl** [eclipse/openj9#12146]
@r30shah : I think it is worth contributing the fix for #11943 to the 0.26 release (openj9-omr repo).  Please let me know if you disagree.

**Comment by r30shah** [comment]
Yes, I agree with it. Let me open up PR Agains the https://github.com/eclipse/openj9-omr/tree/v0.26.0-release

**Comment by r30shah** [comment]
Opened up the PR against openj9-omr/0.26. We can merge it once https://github.com/eclipse/openj9/issues/11943 passes the acceptance.

**Comment by pshipton** [comment]
@0xdaryl As https://github.com/eclipse/openj9-omr/pull/102 is merged, can this be closed?
